src/DssSpell.sol

```diff
@@ -36,40 +55,107 @@ contract DssSpellAction is DssAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
+    uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
 
-    uint256 constant THOUSAND = 10**3;
+    // Math
     uint256 constant MILLION = 10**6;
-    uint256 constant BILLION = 10**9;
+    uint256 constant WAD     = 10**18;
+    uint256 constant RAD     = 10**45;
 
-    // Core Units Budget Addresses
-    address constant INTERIM_MULTISIG = 0x73f09254a81e1F835Ee442d1b3262c1f1d7A13ff;
-    address constant RISK = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
-
-    // Disable Office Hours
-    function officeHours() public override returns (bool) {
-        return false;
-    }
+    // Addresses
+    address constant RWA002_OPERATOR         = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
+    address constant RWA002_GEM              = 0xAAA760c2027817169D7C8DB0DC61A2fb4c19AC23;
+    address constant MCD_JOIN_RWA002_A       = 0xe72C7e90bc26c11d45dBeE736F0acf57fC5B7152;
+    address constant RWA002_A_URN            = 0x225B3da5BE762Ee52B182157E67BeA0b31968163;
+    address constant RWA002_A_INPUT_CONDUIT  = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
+    address constant RWA002_A_OUTPUT_CONDUIT = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
 
     function actions() public override {
-        // Increase the COMP-A Maximum Debt Ceiling from 10M to 30M
-        DssExecLib.setIlkAutoLineDebtCeiling("COMP-A", 30 * MILLION);
-
-        // Increase the ZRX-A Maximum Debt Ceiling from 5M to 10M
-        DssExecLib.setIlkAutoLineDebtCeiling("ZRX-A", 10 * MILLION);
-
-        // Increase the YFI-A Maximum Debt Ceiling from 45M to 75M
-        DssExecLib.setIlkAutoLineDebtCeiling("YFI-A", 75 * MILLION);
-
-        // Increase the PSM-USDC-A Debt Ceiling from 1B to 2B
-        DssExecLib.increaseIlkDebtCeiling("PSM-USDC-A", 1 * BILLION, true);
-
-        // Provide Core Unit Budgets
-
-        // Real-World Finance + Governance (Interim Multi-Sig)
-        DssExecLib.sendPaymentFromSurplusBuffer(INTERIM_MULTISIG, (40 + 80) * THOUSAND);
-
-        // Risk
-        DssExecLib.sendPaymentFromSurplusBuffer(RISK, 100_500);
+        bytes32 ilk   = "RWA002-A";
+        uint256 CEIL  = 5 * MILLION;
+        uint256 PRICE = 5_634_804 * WAD;
```

Comment:
This value needs to be increased to take the `mat` into account. This was missing in the kovan spell. The correct value should be:

```suggestion
        uint256 PRICE = 5_916_544 * WAD;
```

Reply:
5_634_804 is already 2 years of SF discounted by mat. 
Kovan was 5_366_480 (2 years of SF) so I applied mat (105%) on it.

Reply:
@SebVentures you are right - you did include the `mat` buffer already. This is resolved.

Reply:
ACK 

proceeding with numbers as-is